docs/pages/access-controls/teleport-policy/policy-how-to-use.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Teleport Policy usage
```

Comment:
```suggestion
title: Teleport Policy Usage
```
Per the style guide (https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#page-titles)